__tests__/server.test.js

```diff
@@ -87,6 +87,14 @@ describe("backend API project", () => {
           );
         });
     });
+    test("200: that a GET request to /api/articles/artiicle_id responds with an object that includes a comment_count", () => {
```

Comment:
A good start however, we are missing tests for the following
Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999